tests/metrics/base_tests.py

```diff
@@ -141,3 +141,20 @@ def test_client_doesnt_start_collector_thread_in_master_process(is_master_proces
     client = TempStoreClient(server_url="http://example.com", service_name="app_name", secret_token="secret")
     assert mock_start_collect_timer.call_count == before + 1
     client.close()
+
+
+@mock.patch("elasticapm.metrics.base_metrics.DISTINCT_LABEL_LIMIT", 2)
+def test_metric_limit(caplog):
+    m = MetricsSet(MetricsRegistry(0, lambda x: None))
+    with caplog.at_level(logging.WARNING, logger="elasticapm.metrics"):
+        for i in range(4):
+            metric = m.timer("x", some_label=i)
```

Comment:
can you please update the test to show that the other metric types will also be affected?